### PR TITLE
DM-45391: Add scripts to make baseline ComCam curated calibrations. 

### PR DESCRIPTION
**Pull request by erykoff**
These are scripts to (a) average the ITL sensor QE curves; (b) make filter and optics transmission curves based on the official throughputs v1.9.

### CODE REVIEW

File: python/lsst/obs/lsst/script/write_comcam_estimated_transmission_sensor.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+data_root = lsst.utils.getPackageDir("obs_lsst_data")
+
+valid_start = "1970-01-01T00:00:00"
+valid_date = dateutil.parser.parse(valid_start)
+datestr = "".join(re.split(r"[:-]", valid_date.isoformat()))
+
+# Get the transmission_sensor filenames of every ITL detector.
+transmission_sensor_files = []
+for det in lsst_camera:
```
Comment (jchiang87) [inline python/lsst/obs/lsst/script/write_comcam_estimated_transmission_sensor.py:45]: Is it worth putting a comment somewhere in this file stating why the LSSTCam ITL curves are being used? I had to look at the corresponding README in `obs_lsst_data` to make sure I understood what's happening here.

File: python/lsst/obs/lsst/script/write_comcam_estimated_transmission_sensor.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    os.makedirs(out_path, exist_ok=True)
+    out_file = os.path.join(out_path, f"{datestr}.ecsv")
+
+    curve_table.meta.update(
+        {
+            "CALIBDATE": valid_start,
+            "INSTRUME": "ComCam",
+            "OBSTYPE": "transmission_sensor",
+            "TYPE": "transmission_sensor",
+            "DETECTOR": det_num,
```
Comment (jchiang87) [inline python/lsst/obs/lsst/script/write_comcam_estimated_transmission_sensor.py:113]: Should there be a `"SOURCE"` metadata entry somehow indicating that the median of the LSSTCam ITL CCDs was used?

**Review by jchiang87 — CHANGES_REQUESTED** [review]
I think it would be useful to document explicitly that the LSSTCam ITL CCDs are being used for the estimated ComCam QE curves, otherwise all looks good.

**Comment by erykoff** [comment]
@jchiang87 Thanks!  I think this is ready now.

**Review by jchiang87 — APPROVED** [review]
Thanks for the updates!  Looks good.

**Pull request merged**